{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -394,37 +394,34 @@
     'version': 1,
     'disable_existing_loggers': False,
     'filters': {
-        'require_debug_false': {
-            '()': 'django.utils.log.RequireDebugFalse'
-        },
-        'request_id': {
-            '()': 'log_request_id.filters.RequestIDFilter'
-        }
+        'require_debug_false': {'()': 'django.utils.log.RequireDebugFalse'},
```

Comment:
Is there a specific reason, we're not maintaining the indentation here?

I think it was easier to read before.

Reply:
I felt it to be too verbose. The one-liner looked concise.

Reply:
Not very sure about this. My personal preference is to have each option on a different line delimited by a comma so that I don't have to change that line during git diff.

`{
    'one_key': 1,
    'two_key': 2,
}`

But otherwise, it looks good. We can merge this.

Reply:
Me too! :)